src/app/globals.css

```diff
@@ -9,10 +9,14 @@
   --background-color: #eae5f0;
 }
 
-@media (prefers-color-scheme: dark) {
-  :root {
-    --background-color: #1b1228;
-  }
+.dark-mode {
+  --foreground-color: #b077ff;
+  --background-color: #1b1228;
+}
+
+.light-mode {
+  --foreground-color: #b077ff;
+  --background-color: #eae5f0;
 }
```

Comment:
media queries are prefferred when providing css for light vs dark mode instead of having CSS classes hardcoded

this would suffice for example as I have this in my branch right now
```css
@media (prefers-color-scheme: dark) {
  :root {
    --background-color: #1b1228;
    --text-color: #fff;
  }
}
```

be sure to also have the `:root` look like this
```css
:root {
  --max-width: 1100px;
  --border-radius: 12px;
  --font-mono: ui-monospace, Menlo, Monaco, 'Cascadia Mono', 'Segoe UI Mono',
    'Roboto Mono', 'Oxygen Mono', 'Ubuntu Monospace', 'Source Code Pro',
    'Fira Mono', 'Droid Sans Mono', 'Courier New', monospace;

  --foreground-color: #b077ff;
  --background-color: #eae5f0;
  --text-color: #1c1c1c;
}
```

to have the  text color variable too